offer-manager-app/src/main/groovy/life/qbic/portal/offermanager/components/offer/create/CreateOfferViewModel.groovy

```diff
@@ -151,9 +151,9 @@ class CreateOfferViewModel {
         offerCreatedSuccessfully = false
 
         offerId = null
-        projectTitle = ""
-        projectObjective = ""
-        experimentalDesign = ""
+        projectTitle = null
+        projectObjective = null
```

Comment:
They have to be set. If not, it should throw a NullPointer because something went really really wrong.